core/src/edu/asu/c3simulator/screens/MarketingScreen.java

```diff
@@ -0,0 +1,368 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.util.Product;
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Display a list of all products created by the business and give the user the ability to
+ * either market one of them or the company as a whole. The user is also able to choose
+ * from different types of marketing campaigns. This is the first step of two-step wizard.
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class MarketingScreen implements Screen
+{
+	
+	private class productIconListener extends ClickListener
```

Comment:
I have a vague idea of what this is for (and a good understanding after going through the next 30 lines, but a header would be nice. Something brief.